yarn.lock

```diff
@@ -15096,11 +15096,11 @@ preact@10.15.1:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.15.1.tgz#a1de60c9fc0c79a522d969c65dcaddc5d994eede"
   integrity sha512-qs2ansoQEwzNiV5eAcRT1p1EC/dmEzaATVDJNiB3g2sRDWdA7b7MurXdJjB2+/WQktGWZwxvDrnuRFbWuIr64g==
 
-prebid.js@guardian/prebid.js#ddf4251:
+"prebid.js@github:guardian/prebid.js#ddf4251":
```

Comment:
Sorry this is a strange quirk of yarn that flip-flops the syntax for the github reference but this format causes issues for Snyk. Could we update this line to match the 'before' please.